external/js/cothority/src/darc/rules.ts

```diff
@@ -49,9 +91,6 @@ export class Rule extends Message<Rule> {
  */
 export default class Rules extends Message<Rules> {
 
-    static OR = "|";
```

Comment:
I put those here because I'm expecting people to update/add rules from this class and not rule by rule. Is there a good reason to move it ?

Reply:
Well, the `OR` and `AND` are used in the `Rule` class, in the expression, not in `Rules`. Also, this PR proposes a more fine-grained approach to the expressions, not setting whole expressions at once, but adding the possibility to add or remove an identity.